clouddriver-core/src/main/groovy/com/netflix/spinnaker/clouddriver/model/Autoscaler.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2020 Google, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.netflix.spinnaker.clouddriver.model;
+
+import com.netflix.spinnaker.clouddriver.names.NamerRegistry;
+import com.netflix.spinnaker.moniker.Moniker;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+
+public interface Autoscaler {
```

Comment:
I'd like to see class docs on any interface introduced.

Reply:
thanks! added.